src/Symfony/Component/Notifier/Bridge/MsTeams/Action/AbstractMsTeamsActionElement.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MsTeams\Action;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ *
+ * @experimental in 5.3
```

Comment:
```suggestion
 *
 * @experimental in 5.3
```

All of these needs to be removed

Reply:
Are you sure @OskarStark ? Fabpot asked to add in all classes in first comment of the PR